client-cli/src/command.rs

```diff
@@ -112,20 +112,30 @@ impl Command {
                 let fee_algorithm = tendermint_client.genesis()?.fee_policy();
                 let transaction_builder =
                     DefaultTransactionBuilder::new(signer.clone(), fee_algorithm);
+                let transaction_builder2 =
```

Comment:
Is there any reason to create two instances of everything instead of just cloning existing objects wherever needed?

Reply:
it's for life time management,
i'll find a more nice way